guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,308 @@
+
+9. The setup of the ISP Ubuntu on the photos bellow
+
+- `select you language`
+
+![Section9](https://i.imgur.com/Cr4wj1w.png)
+
+- `Press enter on Install Ubuntu Server`
+
+![Section9](https://i.imgur.com/FCEeu3L.png)
+
+- `Select the language again`
+
+![Section9](https://i.imgur.com/cptpprK.png)
+
+- `Select your location`
+
+![Section9](https://i.imgur.com/G03SKC5.png)
+
+- `Press no on the Configure the Keyboard`
+
+![Section9](https://i.imgur.com/6x0pEuX.png)
+
+- `Select Keyboard language`
+
+![Section9](https://i.imgur.com/zQ2EeJk.png)
+
+![Section9](https://i.imgur.com/jsibCL2.png)
+
+- `Select enp0s3 which is the NAT this might vary on your PC`
+
+![Section9](https://i.imgur.com/xN9m2js.png)
+
+- `Set the Hostname of you VM machine to whatever you want`
+
+![Section9](https://i.imgur.com/isHnTuW.png)
+
+- `Set the full name for new user`
+
+![Section9](https://i.imgur.com/SeJZZFJ.png)
+
+![Section9](https://i.imgur.com/mmDpJK9.png)
+
+- `Then set your Username for your account`
+
+![Section9](https://i.imgur.com/zuIqs7E.png)
+
+- `Set the new user password`
+
+![Section9](https://i.imgur.com/t4L1H5v.png)
+
+- `verify password again`
+
+![Section9](https://i.imgur.com/Ypjs1pV.png)
+
+- `Select if you want to encrypt your home directory I advice to select no for this test lab`
+
+![Section9](https://i.imgur.com/Tfju0kx.png)
+
+- `Select your Time zone if the one shown is correct press yes otherwise press no`
+
+![Selection9](https://i.imgur.com/FHvvLvn.png)
+
+- `Select Partition disks select the default the one on the picture.`
+
+![Selection9](https://i.imgur.com/ed9XOYY.png)
+
+- `Select disk partition the default`
+
+![Selection9](https://i.imgur.com/VGm3bD6.png)
+
+- `On the next step select YES on configure LVM`
+
+![Selection9](https://i.imgur.com/btRTEMa.png)
+![Selection9](https://i.imgur.com/K0Xme6H.png)
+
+- `Select the partition size you need to leave it as default`
+
+![Selection9](https://i.imgur.com/TiERK5h.png)
+
+- `Press yes on write the changes to disks`
+
+![Selection9](https://i.imgur.com/eTfOxHK.png)
+
+- `Do not set a proxy leave black and press continue`
+
+![Selection9](https://i.imgur.com/ywLngCl.png)
+
+- `The update question is all up to you. Can be setup to no auto updates or install security updates automatically`
+
+![Selection9](https://i.imgur.com/2MzOrqS.png)
+
+- `On the sofware selection for this lab leave default` 
+
+![Selection9](https://i.imgur.com/325nEqo.png)
+
+- `on the install the grub boot loader question select YES`
+
+![Selection9](https://i.imgur.com/ilxTd3J.png)
+
+- `On the last question select continue`
+
+![Selection9](https://i.imgur.com/az5SGZO.png)
+
+## Third logical Section <a id="Third"></a>
+
+- `After Reboot log in into computer with the username you setup early and enter password to log in`
+
+![Selection9](https://i.imgur.com/HjsqfkV.png)
+![Selection9](https://i.imgur.com/avlp5hu.png)
+
+- `Enable ipv4 routing on command line run` 
+- `(sudo nano /etc/sysctl.conf)`
```

Comment:
This is all a mess lol. Thats why we use code comment. It should read like:

Enable ipv4 routing on Ubuntu. By default Ubuntu doesn't allow for routing on IP traffic. To get this done  you will need to edit `sysctl.conf`. This can be done by using nano like so, `sudo nano /etc/sysctl.conf`. It will require your root password since it effect the entire system when you edit this file.

Reply:
Try to be more verbose with all logical group 3.

Reply:
I did like this because the picture are self explain but i will try to add comments.